content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -94,9 +118,9 @@ Customer Invoices / Vendor Bills`, and click on :guilabel:`⋮` and select :guil
 Matching`, or by going to :menuselection:`Accounting --> Reconciliation`.
 
 .. note::
-   During the reconciliation, if the sum of the debits and credits does not match, it means there is
-   a remaining balance that either needs to be reconciled at a later date or needs to be written off
-   directly.
+   During the :doc:`reconciliation <../../bank/reconciliation/use_cases>`, if the sum of the debits
+   and credits does not match, it means there is a remaining balance that either needs to be
+   reconciled at a later date or needs to be written off directly.
 
 Batch payments matching
```

Comment:
All good, but I'm just wondering why the title here is batch payment, but you don't talk about batch payments, or no link to the batch payments page is added.
So maybe just change the title since you only talk about the synchronization

Reply:
The title is Payments Matching. Batch payment is another paragraph above.
![image](https://user-images.githubusercontent.com/85342080/204286997-3b28afe6-72dd-4779-9766-1cc54149fd37.png)